server/gitrest/package.json

```diff
@@ -21,7 +21,7 @@
     "tsc": "tsc"
   },
   "dependencies": {
-    "@fluidframework/common-utils": "^0.30.0",
+    "@fluidframework/common-utils": "^0.31.0-27017",
```

Comment:
Last I heard from @curtisman, specifying the particular prerelease build number here doesn't play well with other build/release tools.  But it is a handy way to get the lock file correct :)  Unless something's changed I think you need to update all the package.json changes to `-0`.

Reply:
fixed